src/olympia/addons/models.py

```diff
@@ -1129,7 +1129,9 @@ def can_request_review(self):
         latest_version = self.find_latest_version(
             amo.RELEASE_CHANNEL_LISTED, exclude=(amo.STATUS_BETA,))
 
-        return latest_version is not None and latest_version.files.exists()
+        return (latest_version is not None and
+                not latest_version.is_webextension and
+                latest_version.files.exists())
```

Comment:
Isn't that already covered in https://github.com/wagnerand/addons-server/blob/6b72aff448ee23dc42f2680ec0258e8aceadfcdc/src/olympia/addons/models.py#L1123-L1127 ?